lib/taskana-core/src/main/java/pro/taskana/mappings/QueryMapper.java

```diff
@@ -672,8 +672,8 @@
         + "group by WORKBASKET_ID ) AS f where max_read = 1 ) "
         + "</if> "
         + "<if test='taskIds != null'>AND t.ID IN(<foreach item='item' collection='taskIds' separator=',' >#{item}</foreach>)</if> "
-        + "<if test='externalIdIn != null'>AND a.EXTERNAL_ID IN(<foreach item='item' collection='externalIdIn' separator=',' >#{item}</foreach>)</if> "
-        + "<if test='externalIdLike != null'>AND (<foreach item='item' collection='externalIdLike' separator=' OR '>UPPER(a.EXTERNAL_ID) LIKE #{item}</foreach>)</if> "
+        + "<if test='externalIdIn != null'>AND t.EXTERNAL_ID IN(<foreach item='item' collection='externalIdIn' separator=',' >#{item}</foreach>)</if> "
```

Comment:
Changed this because attachment doesn't have such an attribute. So it's probably a typo.
In my opinion it wouldn't make sense to do an own pull request for this.